### PR TITLE
Fix #400

### LINKED ISSUE #400: Another Music Issue (WHEN WILL THIS END)
ok so when an admin does ff2_start_music it plays the song twice at the same time and when the admin does ff2_start_music a 3rd time it plays 3 songs all at once here's a code i tried to add to fix it but made things worst someone help  https://github.com/50DKP/FF2-Official/compare/stable...WakaFloockaFlame:patch-2

### PR DESCRIPTION
**Pull request by shadow93**
Actually fix #400

### CODE REVIEW

**Comment by WakaFloockaFlame** [comment]
@shadow93 Wow your actually Agreeing With me. Thanks shadow :)

**Comment by 50Wliu** [comment]
Can I have a brief explanation of how this fixes the issue?

**Comment by shadow93** [comment]
tl;dr, the currentBGM string was being cleared before StartMusic was executed, which automatically stops any playing BGM. since the string was cleared prior to this, it would render StopMusic unable to stop the current playing BGM, allowing thsi bug in the first place.

**Comment by shadow93** [comment]
TL;DR, the currentBGM string was being blanked out before StopMusic was executed (which StartMusic executes prior to playing a BGM). This in turn, prevented StopMusic from stopping the current BGM that's playing.

**Comment by WakaFloockaFlame** [comment]
@shadow93 Have you tested it does it work? Also your running a fork so make sure it works on 1.10.9

**Comment by shadow93** [comment]
@WakaFloockaFlame, there's always the test server, Whenever it's not one of Wliu's builds, there's one of mine deployed instead. :)

**Comment by WakaFloockaFlame** [comment]
I'm not joining your guys shitty test server i joined it once and it displayed my ip adress when i connected no security at all luckily no 1 was on the server. What type of people are you guys?
